Tests/FeedStoreChallengeTests.swift

```diff
@@ -107,50 +118,55 @@ class FeedStoreChallengeTests: XCTestCase, FeedStoreSpecs {
 //
 //  ***********************
 
-//extension FeedStoreChallengeTests: FailableRetrieveFeedStoreSpecs {
-//
-//	func test_retrieve_deliversFailureOnRetrievalError() throws {
-////		let sut = try makeSUT()
-////
-////		assertThatRetrieveDeliversFailureOnRetrievalError(on: sut)
-//	}
-//
-//	func test_retrieve_hasNoSideEffectsOnFailure() throws {
-////		let sut = try makeSUT()
-////
-////		assertThatRetrieveHasNoSideEffectsOnFailure(on: sut)
-//	}
-//
-//}
+class Cache: Entity {
+	var id: Id = 0
+	var timestamp: Date = Date()
+	// objectbox: backlink = "cache"
+	var feed: ToMany<StoreFeed> = nil
+}
 
-//extension FeedStoreChallengeTests: FailableInsertFeedStoreSpecs {
-//
-//	func test_insert_deliversErrorOnInsertionError() throws {
-////		let sut = try makeSUT()
-////
-////		assertThatInsertDeliversErrorOnInsertionError(on: sut)
-//	}
-//
-//	func test_insert_hasNoSideEffectsOnInsertionError() throws {
-////		let sut = try makeSUT()
-////
-////		assertThatInsertHasNoSideEffectsOnInsertionError(on: sut)
-//	}
-//
-//}
+class StoreFeed: Entity {
+	var id: Id = 0
+	var modelId: String = ""
+	var description: String?
+	var location: String?
+	var url: String = ""
+	var cache: ToOne<Cache> = nil
+}
 
-//extension FeedStoreChallengeTests: FailableDeleteFeedStoreSpecs {
-//
-//	func test_delete_deliversErrorOnDeletionError() throws {
-////		let sut = try makeSUT()
-////
-////		assertThatDeleteDeliversErrorOnDeletionError(on: sut)
-//	}
-//
-//	func test_delete_hasNoSideEffectsOnDeletionError() throws {
-////		let sut = try makeSUT()
-////
-////		assertThatDeleteHasNoSideEffectsOnDeletionError(on: sut)
-//	}
-//
-//}
+extension FeedStoreChallengeTests: FailableRetrieveFeedStoreSpecs {
+	
+	private func addInvalidFormattedFeedTo(storeURL: URL) throws {
+		let store = try Store(directoryPath: storeURL.path)
+		let cache = Cache()
+		cache.timestamp = Date()
+
+		let storeFeed = StoreFeed()
+		storeFeed.modelId = "Invalid UUID"
+		storeFeed.description = "A description"
+		storeFeed.location = "A location"
+		storeFeed.url = "Invalid URL"
+		storeFeed.cache.target = cache
+
+		try! store.box(for: StoreFeed.self).put(storeFeed)
```

Comment:
This method can throw, so you shouldn't force-try with `try!`. 

This way, if there's an error, it'll generate a test failure instead of a crash.

```suggestion
		try store.box(for: StoreFeed.self).put(storeFeed)
```